docs/source/api.rst

```diff
@@ -19,11 +19,18 @@ Label Encoding
  .. autoclass:: cuml.preprocessing.LabelEncoder
     :members:
 
-Dataset Generation
-------------------
+Dataset Generation (Single-GPU)
+-------------------------------
 
   .. automethod:: cuml.datasets.make_blobs
 
+
+Dataset Generation (Dask)
```

Comment:
```suggestion
Dataset Generation (Dask based Multi-GPU)
```
I think adding multi-gpu explicitly for symmetry with the Single-GPU one would be a good idea. Open question: shouldn't this be in the mnmg section?

Reply:
I debated whether to put it in the multi-gpu section... I thought it was farther away then from the other data gen parts and may be more confusing then. Agreed about the renaming to Dask-based multi GPU

Reply:
yeah, I think it might be a good idea to leave it here and depending of if we get questions or feedback we could move it, but after looking at the docs, the placement here looks fine to me